checker/bin/wpi-annotation-paths.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Given the name of a directory containing the results produced by an
+# invocation of wpi-many.sh, output the paths to the .ajava files to stdout.
```

Comment:
This description suggests that the output will be **just** the paths (in the style of e.g., `find`), but the script actually produces formatted output that's designed for a human to read. I think the formatted output is nice, so I recommend just adding "in human-readable format" or something similar to this comment. 

Alternatively, if we wanted to make this script useful for both scripting and for humans, we could add a flag to control whether just the paths are printed (i.e., in suitable form for another tool to consume) or whether the current human-readable output format is used. That's probably over-engineering, though, unless you're expecting to immediately use this script as part of a pipeline.

Reply:
I'll add the "in human-readable format,"

There isn't yet a use-case for having the paths printed in another format, but I'll open another PR if that becomes the case. Thanks Martin!